<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Taxonomy of <em>Gammaproteobacteria</em> • yatah</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/simplex/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Taxonomy of &lt;em&gt;Gammaproteobacteria&lt;/em&gt;">
<meta property="og:description" content="">
<meta property="og:image" content="https://abichat.github.io/yatah/logo.png">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-131161788-01"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-131161788-01');
</script>
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">yatah</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.0.0.9006</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home"></span>
     
  </a>
</li>
<li>
  <a href="../articles/gammaproteobacteria.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="../news/index.html">
    <span class="fa fa-newspaper-o"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/abichat/yatah">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>Taxonomy of <em>Gammaproteobacteria</em>
</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/abichat/yatah/blob/master/vignettes/gammaproteobacteria.Rmd"><code>vignettes/gammaproteobacteria.Rmd</code></a></small>
      <div class="hidden name"><code>gammaproteobacteria.Rmd</code></div>

    </div>

    
    
<div id="preliminaries" class="section level2">
<h2 class="hasAnchor">
<a href="#preliminaries" class="anchor"></a>Preliminaries</h2>
<p>This vignette illustrates the most useful functions of <strong>yatah</strong>.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(curatedMetagenomicData) <span class="co"># from Bioconductor</span></a>
<a class="sourceLine" id="cb1-2" data-line-number="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(ggtree) <span class="co"># from Bioconductor</span></a>
<a class="sourceLine" id="cb1-3" data-line-number="3"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(dplyr)</a>
<a class="sourceLine" id="cb1-4" data-line-number="4"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(yatah)</a></code></pre></div>
</div>
<div id="data" class="section level2">
<h2 class="hasAnchor">
<a href="#data" class="anchor"></a>Data</h2>
<p>For this example, we use normalized data given in <strong>curatedMetagenomicData</strong> (<span class="citation">Pasolli et al. (<a href="#ref-cmd2017">2017</a>)</span>) corresponding to the dataset from <span class="citation">Zeller et al. (<a href="#ref-zeller2014potential">2014</a>)</span>. We collect the lineages of bacteria present in 199 stool samples.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1">taxonomy &lt;-</a>
<a class="sourceLine" id="cb2-2" data-line-number="2"><span class="st">  "ZellerG_2014.metaphlan_bugs_list.stool"</span> <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-3" data-line-number="3"><span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/curatedMetagenomicData/topics/curatedMetagenomicData">curatedMetagenomicData</a></span>(<span class="dt">dryrun =</span> <span class="ot">FALSE</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-4" data-line-number="4"><span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/curatedMetagenomicData/topics/mergeData">mergeData</a></span>() <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-5" data-line-number="5"><span class="st">  </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/tidyeval.html">exprs</a></span>() <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb2-6" data-line-number="6"><span class="st">  </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/reexports.html">as_tibble</a></span>(<span class="dt">rownames =</span> <span class="st">"lineages"</span>) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb2-7" data-line-number="7"><span class="st">  </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/select.html">select</a></span>(lineages)</a></code></pre></div>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1">taxonomy</a>
<a class="sourceLine" id="cb3-2" data-line-number="2"><span class="co">#&gt; # A tibble: 1,585 x 1</span></a>
<a class="sourceLine" id="cb3-3" data-line-number="3"><span class="co">#&gt;    lineages                                  </span></a>
<a class="sourceLine" id="cb3-4" data-line-number="4"><span class="co">#&gt;    &lt;chr&gt;                                     </span></a>
<a class="sourceLine" id="cb3-5" data-line-number="5"><span class="co">#&gt;  1 k__Bacteria                               </span></a>
<a class="sourceLine" id="cb3-6" data-line-number="6"><span class="co">#&gt;  2 k__Viruses                                </span></a>
<a class="sourceLine" id="cb3-7" data-line-number="7"><span class="co">#&gt;  3 k__Bacteria|p__Firmicutes                 </span></a>
<a class="sourceLine" id="cb3-8" data-line-number="8"><span class="co">#&gt;  4 k__Bacteria|p__Bacteroidetes              </span></a>
<a class="sourceLine" id="cb3-9" data-line-number="9"><span class="co">#&gt;  5 k__Bacteria|p__Actinobacteria             </span></a>
<a class="sourceLine" id="cb3-10" data-line-number="10"><span class="co">#&gt;  6 k__Bacteria|p__Verrucomicrobia            </span></a>
<a class="sourceLine" id="cb3-11" data-line-number="11"><span class="co">#&gt;  7 k__Bacteria|p__Proteobacteria             </span></a>
<a class="sourceLine" id="cb3-12" data-line-number="12"><span class="co">#&gt;  8 k__Viruses|p__Viruses_noname              </span></a>
<a class="sourceLine" id="cb3-13" data-line-number="13"><span class="co">#&gt;  9 k__Bacteria|p__Candidatus_Saccharibacteria</span></a>
<a class="sourceLine" id="cb3-14" data-line-number="14"><span class="co">#&gt; 10 k__Bacteria|p__Firmicutes|c__Clostridia   </span></a>
<a class="sourceLine" id="cb3-15" data-line-number="15"><span class="co">#&gt; # … with 1,575 more rows</span></a></code></pre></div>
</div>
<div id="filtering" class="section level2">
<h2 class="hasAnchor">
<a href="#filtering" class="anchor"></a>Filtering</h2>
<p>Here, we have all the present bacteria at all different ranks. As we are just interested in genera that belong to the <em>Gammaproteobacteria</em> class, we <code><a href="https://dplyr.tidyverse.org/reference/filter.html">filter()</a></code> the lineages with <code><a href="../reference/is_clade.html">is_clade()</a></code> and <code><a href="../reference/is_rank.html">is_rank()</a></code>. The genus name is accessible with <code><a href="../reference/last_clade.html">last_clade()</a></code>.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1">taxonomy <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb4-2" data-line-number="2"><span class="st">  </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/filter.html">filter</a></span>(<span class="kw"><a href="../reference/is_clade.html">is_clade</a></span>(lineages, <span class="st">"Gammaproteobacteria"</span>),</a>
<a class="sourceLine" id="cb4-3" data-line-number="3">         <span class="kw"><a href="../reference/is_rank.html">is_rank</a></span>(lineages, <span class="st">"genus"</span>)) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb4-4" data-line-number="4"><span class="st">  </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/mutate.html">mutate</a></span>(<span class="dt">genus =</span> <span class="kw"><a href="../reference/last_clade.html">last_clade</a></span>(lineages))</a>
<a class="sourceLine" id="cb4-5" data-line-number="5"><span class="co">#&gt; # A tibble: 26 x 2</span></a>
<a class="sourceLine" id="cb4-6" data-line-number="6"><span class="co">#&gt;    lineages                                            genus               </span></a>
<a class="sourceLine" id="cb4-7" data-line-number="7"><span class="co">#&gt;    &lt;chr&gt;                                               &lt;chr&gt;               </span></a>
<a class="sourceLine" id="cb4-8" data-line-number="8"><span class="co">#&gt;  1 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Escherichia         </span></a>
<a class="sourceLine" id="cb4-9" data-line-number="9"><span class="co">#&gt;  2 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Haemophilus         </span></a>
<a class="sourceLine" id="cb4-10" data-line-number="10"><span class="co">#&gt;  3 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Enterobacteriaceae_…</span></a>
<a class="sourceLine" id="cb4-11" data-line-number="11"><span class="co">#&gt;  4 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Pseudomonas         </span></a>
<a class="sourceLine" id="cb4-12" data-line-number="12"><span class="co">#&gt;  5 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Enterobacter        </span></a>
<a class="sourceLine" id="cb4-13" data-line-number="13"><span class="co">#&gt;  6 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Aggregatibacter     </span></a>
<a class="sourceLine" id="cb4-14" data-line-number="14"><span class="co">#&gt;  7 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Hafnia              </span></a>
<a class="sourceLine" id="cb4-15" data-line-number="15"><span class="co">#&gt;  8 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Actinobacillus      </span></a>
<a class="sourceLine" id="cb4-16" data-line-number="16"><span class="co">#&gt;  9 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Sinobacteraceae_unc…</span></a>
<a class="sourceLine" id="cb4-17" data-line-number="17"><span class="co">#&gt; 10 k__Bacteria|p__Proteobacteria|c__Gammaproteobacter… Citrobacter         </span></a>
<a class="sourceLine" id="cb4-18" data-line-number="18"><span class="co">#&gt; # … with 16 more rows</span></a></code></pre></div>
</div>
<div id="taxonomic-table" class="section level2">
<h2 class="hasAnchor">
<a href="#taxonomic-table" class="anchor"></a>Taxonomic table</h2>
<p>It is useful to have a taxonomic table. <code><a href="../reference/taxtable.html">taxtable()</a></code> do the job.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1">gammap_table &lt;-</a>
<a class="sourceLine" id="cb5-2" data-line-number="2"><span class="st">  </span>taxonomy <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb5-3" data-line-number="3"><span class="st">  </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/filter.html">filter</a></span>(<span class="kw"><a href="../reference/is_clade.html">is_clade</a></span>(lineages, <span class="st">"Gammaproteobacteria"</span>),</a>
<a class="sourceLine" id="cb5-4" data-line-number="4">         <span class="kw"><a href="../reference/is_rank.html">is_rank</a></span>(lineages, <span class="st">"genus"</span>)) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb5-5" data-line-number="5"><span class="st">  </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/pull.html">pull</a></span>(lineages) <span class="op">%&gt;%</span><span class="st"> </span></a>
<a class="sourceLine" id="cb5-6" data-line-number="6"><span class="st">  </span><span class="kw"><a href="../reference/taxtable.html">taxtable</a></span>()</a></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1"><span class="kw"><a href="https://dplyr.tidyverse.org/reference/reexports.html">as_tibble</a></span>(gammap_table)</a>
<a class="sourceLine" id="cb6-2" data-line-number="2"><span class="co">#&gt; # A tibble: 26 x 6</span></a>
<a class="sourceLine" id="cb6-3" data-line-number="3"><span class="co">#&gt;    kingdom  phylum     class        order       family      genus          </span></a>
<a class="sourceLine" id="cb6-4" data-line-number="4"><span class="co">#&gt;    &lt;chr&gt;    &lt;chr&gt;      &lt;chr&gt;        &lt;chr&gt;       &lt;chr&gt;       &lt;chr&gt;          </span></a>
<a class="sourceLine" id="cb6-5" data-line-number="5"><span class="co">#&gt;  1 Bacteria Proteobac… Gammaproteo… Enterobact… Enterobact… Escherichia    </span></a>
<a class="sourceLine" id="cb6-6" data-line-number="6"><span class="co">#&gt;  2 Bacteria Proteobac… Gammaproteo… Pasteurell… Pasteurell… Haemophilus    </span></a>
<a class="sourceLine" id="cb6-7" data-line-number="7"><span class="co">#&gt;  3 Bacteria Proteobac… Gammaproteo… Enterobact… Enterobact… Enterobacteria…</span></a>
<a class="sourceLine" id="cb6-8" data-line-number="8"><span class="co">#&gt;  4 Bacteria Proteobac… Gammaproteo… Pseudomona… Pseudomona… Pseudomonas    </span></a>
<a class="sourceLine" id="cb6-9" data-line-number="9"><span class="co">#&gt;  5 Bacteria Proteobac… Gammaproteo… Enterobact… Enterobact… Enterobacter   </span></a>
<a class="sourceLine" id="cb6-10" data-line-number="10"><span class="co">#&gt;  6 Bacteria Proteobac… Gammaproteo… Pasteurell… Pasteurell… Aggregatibacter</span></a>
<a class="sourceLine" id="cb6-11" data-line-number="11"><span class="co">#&gt;  7 Bacteria Proteobac… Gammaproteo… Enterobact… Enterobact… Hafnia         </span></a>
<a class="sourceLine" id="cb6-12" data-line-number="12"><span class="co">#&gt;  8 Bacteria Proteobac… Gammaproteo… Pasteurell… Pasteurell… Actinobacillus </span></a>
<a class="sourceLine" id="cb6-13" data-line-number="13"><span class="co">#&gt;  9 Bacteria Proteobac… Gammaproteo… Xanthomona… Sinobacter… Sinobacteracea…</span></a>
<a class="sourceLine" id="cb6-14" data-line-number="14"><span class="co">#&gt; 10 Bacteria Proteobac… Gammaproteo… Enterobact… Enterobact… Citrobacter    </span></a>
<a class="sourceLine" id="cb6-15" data-line-number="15"><span class="co">#&gt; # … with 16 more rows</span></a></code></pre></div>
</div>
<div id="taxonomic-tree" class="section level2">
<h2 class="hasAnchor">
<a href="#taxonomic-tree" class="anchor"></a>Taxonomic tree</h2>
<p>To have a tree, use <code><a href="../reference/taxtree.html">taxtree()</a></code> with a taxonomic table in input. By default, it collapses ranks with only one subrank.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">gammap_tree &lt;-<span class="st"> </span><span class="kw"><a href="../reference/taxtree.html">taxtree</a></span>(gammap_table)</a>
<a class="sourceLine" id="cb7-2" data-line-number="2">gammap_tree</a>
<a class="sourceLine" id="cb7-3" data-line-number="3"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb7-4" data-line-number="4"><span class="co">#&gt; Phylogenetic tree with 26 tips and 7 internal nodes.</span></a>
<a class="sourceLine" id="cb7-5" data-line-number="5"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb7-6" data-line-number="6"><span class="co">#&gt; Tip labels:</span></a>
<a class="sourceLine" id="cb7-7" data-line-number="7"><span class="co">#&gt;  Escherichia, Enterobacteriaceae_noname, Enterobacter, Hafnia, Citrobacter, Pantoea, ...</span></a>
<a class="sourceLine" id="cb7-8" data-line-number="8"><span class="co">#&gt; Node labels:</span></a>
<a class="sourceLine" id="cb7-9" data-line-number="9"><span class="co">#&gt;  Gammaproteobacteria, Enterobacteriaceae, Pasteurellaceae, Pseudomonadales, Moraxellaceae, Xanthomonadales, ...</span></a>
<a class="sourceLine" id="cb7-10" data-line-number="10"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb7-11" data-line-number="11"><span class="co">#&gt; Rooted; includes branch lengths.</span></a></code></pre></div>
<p>Instead of a classical <code>plot</code>, we use <strong>ggtree</strong> (<span class="citation">Yu et al. (<a href="#ref-ggtree2017">2017</a>)</span>) to display the tree.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/ggtree/topics/ggtree">ggtree</a></span>(gammap_tree) <span class="op">+</span></a>
<a class="sourceLine" id="cb8-2" data-line-number="2"><span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/ggtree/topics/geom_tiplab">geom_tiplab</a></span>(<span class="dt">hjust =</span> <span class="dv">1</span>, <span class="dt">geom =</span> <span class="st">"label"</span>) <span class="op">+</span></a>
<a class="sourceLine" id="cb8-3" data-line-number="3"><span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/ggtree/topics/geom_nodelab">geom_nodelab</a></span>(<span class="dt">hjust =</span> <span class="dv">0</span>, <span class="dt">size =</span> <span class="dv">3</span>)</a></code></pre></div>
<p><img src="gammaproteobacteria_files/figure-html/ggtree-1.png" width="672"></p>
</div>
<div id="references" class="section level2 unnumbered">
<h2 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h2>
<div id="refs" class="references">
<div id="ref-cmd2017">
<p>Pasolli, Edoardo, Lucas Schiffer, Paolo Manghi, Audrey Renson, Valerie Obenchain, Duy Tin Truong, Francesco Beghini, et al. 2017. “Accessible, Curated Metagenomic Data Through Experimenthub.” <em>Nat. Methods</em> 14 (11): 1023–4. <a href="https://doi.org/10.1038/nmeth.4468" class="uri">https://doi.org/10.1038/nmeth.4468</a>.</p>
</div>
<div id="ref-ggtree2017">
<p>Yu, Guangchuang, David Smith, Huachen Zhu, Yi Guan, and Tommy Tsan-Yuk Lam. 2017. “Ggtree: An R Package for Visualization and Annotation of Phylogenetic Trees with Their Covariates and Other Associated Data.” <em>Methods in Ecology and Evolution</em> 8 (1): 28–36. <a href="https://doi.org/10.1111/2041-210X.12628" class="uri">https://doi.org/10.1111/2041-210X.12628</a>.</p>
</div>
<div id="ref-zeller2014potential">
<p>Zeller, Georg, Julien Tap, Anita Y Voigt, Shinichi Sunagawa, Jens Roat Kultima, Paul I Costea, Aurélien Amiot, et al. 2014. “Potential of Fecal Microbiota for Early-Stage Detection of Colorectal Cancer.” <em>Molecular Systems Biology</em> 10 (11). EMBO Press: 766.</p>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#preliminaries">Preliminaries</a></li>
      <li><a href="#data">Data</a></li>
      <li><a href="#filtering">Filtering</a></li>
      <li><a href="#taxonomic-table">Taxonomic table</a></li>
      <li><a href="#taxonomic-tree">Taxonomic tree</a></li>
      <li><a href="#references">References</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by <a href="https://abichat.github.io">Antoine Bichat</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
